apps/speed.c

```diff
@@ -155,8 +155,12 @@ static int usertime = 1;
 
 static double Time_F(int s);
 static void print_message(const char *s, long num, int length, int tm);
+#if (!defined(OPENSSL_NO_RSA) && !defined(OPENSSL_NO_DEPRECATED_3_0)) \
```

Comment:
In spite of the existing OPENSSL_NO_RSA guards in this file and elsewhere, `no-rsa` is not a valid option

Reply:
I know. I left `OPENSSL_NO_RSA` in for consistency with the rest of `speed.c`.